recipe_scrapers/maangchi.py

```diff
@@ -8,7 +10,9 @@ def host(cls):
         return "maangchi.com"
 
     def ingredients(self):
-        before = self.soup.find("h2", string="Ingredients").find_all_next("li")
+        before = self.soup.find(
+            "h2", string=re.compile(r"Ingredients(\s*\(.*\))?")
+        ).find_all_next("li")
```

Comment:
We might be able to simplify this by removing the bracket-matching: `BeautifulSoup`'s [filtering](https://beautiful-soup-4.readthedocs.io/en/latest/#kinds-of-filters) behaves differently for plain strings (`"Ingredients"`) compared to regular expressions (`re.compile(r"Ingredients")`).

(plain strings exact-match on entire element content; regular expressions are searched for the pattern anywhere within the content)

```suggestion
        before = self.soup.find(
            "h2", string=re.compile(r"Ingredients")
        ).find_all_next("li")
```

Reply:
I didn't know this, good to know!

Reply:
You're welcome!